lib/time-uuid.js

```diff
@@ -99,18 +114,7 @@ TimeUuid.max = function (date, ticks) {
  * If not provided a random clockId will be generated.
  */
 TimeUuid.now = function (nodeId, clockId) {
-    return new TimeUuid(null, null, nodeId, clockId);
-};
-
-var uuidRe = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/;
-
-/**
- * Tests if a given string is a valid v1 UUID
- * @param s {String} a uuid string to test.
- * @returns {boolean} true if the given string is a v1 UUID, false otherwise
- */
-TimeUuid.test = function(s) {
-    return uuidRe.test(s);
```

Comment:
We still need this function. We've added it to the upstream library for convinience